assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -1214,4 +1214,22 @@ describe("DiversionPage", () => {
       expect(missedStopIcon.getAll(container)).toHaveLength(1)
     })
   })
+
+  describe("'Change route or direction' button", () => {
+    /*
+     * This test is here because this button is now part of one of the
+     * components, and we want to make sure that we don't accidentally
+     * render it when we didn't intend to.
+     */
+    test("the button doesn't render yet in context", async () => {
```

Comment:
This is kind of silly but useful still. https://github.com/mbta/skate/pull/2625 adds another test that makes this whole thing less silly, although I'm not convinced that we should actually merge https://github.com/mbta/skate/pull/2625, since it doesn't really do anything worthwhile.